scripts/tools/initialize_virtualenv.py

```diff
@@ -0,0 +1,136 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We should add shebang here  (see other scripts):

```
#!/usr/bin/env python3
```

And make the script executable.